### DISCUSSION TITLE
滑块验证码问题

### DISCUSSION THREAD

**Discussion by chen2gou** [simple-robot/Discussions#180] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

登录场景：

1. TxCaptchaHelper 无效
2. f12 获取ticket 报45需要等mirai更新
3. mirai-login-solver-sakura机器人不支持集成

所以现在机器人没法用了是吗，现在是否有其他解决方案？

**Comment by ForteScarlet** [comment]
**似乎**使用手表协议扫码登录有概率可以使用。（我没有亲自验证过）以及“机器人不支持集成”是什么意思？按理说只要mirai-core能够集成的东西simbot–mirai就可以集成。

不过目前来看mirai已经基本不再更新，其他可用的第三方框架也屈指可数了。
但是**似乎**官方的api再有一阵子就可能会开放，所以我们正在等待官方的api公开问世。至于这“一阵子”到底有多久我就不知道了。🥲
simbot除了mirai的其他几个组件都是正常可用的状态，实在不行也可以选择去用QQ频道😘
